### DISCUSSION TITLE
Automatically creating many from a O2M relationship by creating the one

### DISCUSSION THREAD

**Discussion by YannickMol** [directus/directus#12184] [Archive]
Let's say I have a 'Project' collection and a 'Milestones' collection with an O2M relationship where multiple milestones can belong to one project. Is there currently a possibility to create a certain set of milestones (which is preset) automatically when creating a project?

**Comment by rijkvanzanten** [comment]
You should be able to configure the Preset for the role to include the to-be-created items for that o2m field nested 🤔
